build/docker/ubuntu-jammy/Dockerfile

```diff
@@ -251,10 +251,13 @@ RUN apt-get install -yq \
   libpython2-dev \
   libxml2-dev && \
   cd / && \
-  wget --quiet https://download.swift.org/swift-5.7-release/ubuntu2204/swift-5.7-RELEASE/swift-5.7-RELEASE-ubuntu22.04.tar.gz && \
-  tar xf swift-5.7-RELEASE-ubuntu22.04.tar.gz --strip-components=1 && \
-  rm swift-5.7-RELEASE-ubuntu22.04.tar.gz && \
-  swift --version
+  wget --quiet https://download.swift.org/swift-5.7-release/ubuntu2004/swift-5.7-RELEASE/swift-5.7-RELEASE-ubuntu20.04.tar.gz && \
```

Comment:
Is 20.04 jammy?

Reply:
oh good catch, 22.04 is jammy, 20.04 is focal